ci/params.py

```diff
@@ -19,6 +19,11 @@ class AllParams:
         name='branch',
         description='branch of git repository',
     )
+    build_dict_str = tkn.model.NamedParam(
+        name='build_dict_str',
```

Comment:
naming-nit: maybe rather name this `build_dict_json`? or maybe something more expressive, such as `build_tasks_to_manifest_keys_json?